script/hassfest/zeroconf.py

```diff
@@ -45,16 +61,50 @@ def generate_and_validate(integrations: Dict[str, Integration]):
             continue
 
         for service_type in service_types:
+            service_type_dict[service_type].append(domain)
 
-            if service_type not in service_type_dict:
-                service_type_dict[service_type] = []
+        for model in homekit_models:
+            # We add a space, as we want to test for it to be model + space.
```

Comment:
Don't we want as proper keys as possible and do this logic in the later components?

Reply:
Well we specify if a model starts with "X ", so this is what we are testing for. I see your point but it means more calculation while running, instead of during build time.